Formula/openssh.rb

```diff
@@ -53,9 +57,46 @@ def install
     # potential to break scripts, so recreate it for now.
     # Debian have done the same thing.
     bin.install_symlink bin/"ssh" => "slogin"
+
+    # https://opensource.apple.com/source/OpenSSH/OpenSSH-209.30.4/com.openssh.sshd.sb
```

Comment:
We're already dependent on that site for bsdmake, cctools-headers, cctools, cvs, dyld-headers, ksh, ld64, libunwind-headers, lsyncd, md, procmail, and rpcgen, so what's one more? Yeah, I'd prefer a resource block to a heredoc.